__tests__/api.test.js

```diff
@@ -302,4 +388,12 @@ describe('GET /api/users', () => {
               ])
         })
     });
+});
+
+describe('Delete comment by id', () => {
+    test('returns 204', () => {
+        return request(app)
+        .delete('/api/comments/1')
+        .expect(204)
+    });
```

Comment:
The same error tests will need to be applied here too
- 404 - article id not found
- 400 - article id invalid